src/worker.js

```diff
@@ -133,6 +133,7 @@ class Worker {
     return Promise.fromCallback((cb) => {
       cls.run(() => {
         cls.set('tid', this.tid)
+        cls.set('previousEvent', this.queue)
```

Comment:
I like just `WorkerName` since in the scope of the function there is no `next` it is always current.